src/pages/ProductList/ProductList.js

```diff
@@ -95,14 +127,17 @@ class ProductList extends Component {
           </section>
           <div className="products">
             <div className="productsLine">
-              {searchStyle.map(data => {
+              {searchStyle.map((data, idx) => {
```

Comment:
```suggestion
              {searchStyle.map((data, idx) => {
                const [productPic, productPicReverse] = data.img_url
```
객체 뿐만 아니라 배열도 구조분해 할당이 가능하기 때문에 이렇게 변수명을 지어주시면 0, 1 같은 인덱스 말고 이름으로 데이터를 구분할 수 있을거 같아요

Reply:
🐊 수정 완료하였습니다.